src/components/Form.js

```diff
@@ -26,7 +27,18 @@ const styles = {
 		maxWidth: '33%',
 		width: '100%',
 		marginTop: 16,
-		alignSelf: 'flex-end',
+		alignSelf: 'center',
+		border: '2px solid black',
+		borderRadius: 0,
```

Comment:
You shouldn't need to define this unless there's something weird going on. `borderRadius` defaults to 0.